dist/exponential.go

```diff
@@ -229,8 +218,26 @@ func (e Exponential) Survival(x float64) float64 {
 	return math.Exp(-e.Rate * x)
 }
 
-// UnmarshalParameters implements the ParameterMarshaler interface
-func (e *Exponential) UnmarshalParameters(p []Parameter) {
+// Variance returns the variance of the probability distribution.
+func (e Exponential) Variance() float64 {
+	return 1 / (e.Rate * e.Rate)
+}
+
+// getParameters implements the parameterMutator interface
+func (e Exponential) getParameters(p []Parameter) []Parameter {
```

Comment:
Shouldn't this just be "parameters"? Go generally doesn't use "Get" unless otherwise necessary.